PresentData/PresentMonTraceConsumer.cpp

```diff
@@ -707,6 +717,12 @@ void PMTraceConsumer::HandleDXGKEvent(EVENT_RECORD* pEventRecord)
         HandleDxgkBlt(hdr, hwnd, bRedirectedPresent);
         break;
     }
+    case Microsoft_Windows_DxgKrnl::Blit_Cancel::Id:
+    {
+        TRACK_PRESENT_PATH_GENERATE_ID();

```

Comment:
This should be a TRACK_PRESENT_PATH() macro, instead of ..._GENERATE_ID().  I know it's sort of confusing... if there's any improvement to the comment on line 37 you can think of let me know :)

Reply:
Thanks, I hadn't seen the comment before... this makes more sense now.